README.md

```diff
@@ -11,36 +11,30 @@ This documentation is about how to use the API from Gmail and providing a CSV fi
 
 Firts clone or download the repo with the [git commands](https://confluence.atlassian.com/bitbucket/clone-a-repository-223217891.html)
 
-After that open the file on your text editor and execute this on the terminal
-
-Then, go to the official page of [GMAIL API](https://developers.google.com/gmail/api/quickstart/ruby) to obtain your credentials.json
+Then, in order to use the Gmail API you will need to create a project into your Google console. You can easily do it throght the following Gmail documentation [GMAIL API](https://developers.google.com/gmail/api/quickstart/ruby). This will give you your credentials.json.
 
 Follow Step 1 to turn on the Gmail API and don't forget to download your client configuration (your file probably will be on the download's folder, please change it at the right folder)
 
 Step 2 you need to Install the Google Client Library. Verify that you are on the right path and add this line to your terminal to add the gem
 ```bash
 gem install google-api-client
 ```
-Execute this
+As the application purpose is to create a CSV, you need to have the csv gem, if you don't have it, then you can install it executing:
 ```bash
 gem install 'csv'
 ```
-After that execute this on the terminal
+Time to run the program:
 ```bash
-ruby nameofyourfile.rb
+ruby app.rb
 ```
 
-The first time you run the project, it going to prompt you to authorize access:
-
-- Click on the link that gives you and this going to open a new window or tab in your browser. It this fails, don't worry about it just copy the URL on your browser
+The first time you execute the file, you will be asked to authorize access:
 
-- Then Google will be prompted to log in in case you are not. If you are logged into differents Gmail accounts, Google will ask you to select one account to use for the authorization.
+- You can copy the given link and paste it into your favorite explorer. Then Google will be prompted to log in, in case you are not, and if you are logged into differents Gmail accounts, Google will ask you to select one account to use for the authorization. Here you should grant the right permisions to the application.
 
 - Click the Accept button.
 
-- Copy the code on your terminal 
-
-Now you may close the window/tab.
+- Copy the given code and paste it into your terminal. At this point you can close the explorer tab.
```

Comment:
close the `browser` tab. (Here again, I think the word can be changed to something different to avoid confusion.)